contracts/src/base/registry/facets/distribution/RewardsDistribution.sol

```diff
@@ -96,6 +96,33 @@ contract RewardsDistribution is
     );
   }
 
+  function operatorClaimByAddress(address operatorAddress) external {
+    RewardsDistributionStorage.Layout storage ds = RewardsDistributionStorage
+      .layout();
+
+    NodeOperatorStorage.Layout storage nos = NodeOperatorStorage.layout();
+    if (nos.claimerByOperator[operatorAddress] != msg.sender)
```

Comment:
Can this be put more idiomatically in a function modifier `onlyClaimerByOperator` ?